post_installation_configuration/deploy-heterogeneous-configuration.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+:context: deploy-heterogeneous-configuration
+[id="post-install-deploy-heterogeneous-configuration"]
+= Configuring a heterogeneous cluster
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+A heterogeneous cluster is a cluster that supports worker machines with different architectures. You can deploy a heterogeneous cluster by creating an Azure installer-provisioned cluster using the heterogeneous installer binary. For Azure installation, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing on Azure with customizations].
+
+The following procedures explain how to generate a `arm64` bootimage and create an Azure machine set with the `arm64` bootimage. This will add `arm64` worker nodes to your heterogeneous cluster and deploy a ARM64 virtual machine (VM). Heterogeneous clusters are only available on Azure installer-provisioned infrastructures with `x86_64` control planes. 
```

Comment:
`and deploy a ARM64 virtual machine` => `and deploy the desired number of ARM64 virtual machines` (see the `replicas` field in the MachineSet example above)